checkstyle-tester/README.md

```diff
@@ -63,15 +63,18 @@ will be analyzed by Checkstyle during report generation.
 This option is useful for Windows users where they are restricted to maximum directory depth
 (optional, default is false).
 
-**extraMvnRegressionOptions** (xm) - this option can be used to supply extra command line arguments
-to maven during the Checkstyle regression run.  For example, if you want to skip site generation, you
-would add `--extraMvnRegressionOptions "-Dmaven.site.skip=true"` to `diff.groovy` execution.
+**extraRegressionOptions** (xo) - this option can be used to supply extra command line arguments
+to Checkstyle during the regression run.
 
 **allowExcludes** (g) - this option tells `diff.groovy` to allow paths and files defined in the file
  specified for the `--listOfProjects (-l)` argument to be excluded from report generation
  (optional, default is false). This option is
  used for files that are not compilable or that Checkstyle cannot parse.
 
+**continueOnError** (ce) - whether to fail or continue the Checkstyle' run when it reports a
```

Comment:
`Checkstyle'` -> `Checkstyle`

Reply:
Why do we need this option we are using haltOnException = false and severity = warning?

Reply:
> Why do we need this option we are using haltOnException = false and severity = warning?

See https://github.com/checkstyle/contribution/pull/747#discussion_r1059673349

Reply:
We do have one config used in main repo for regression that has an error set.
https://github.com/checkstyle/checkstyle/blob/d3a5f9a5ba4a1f2df28900dd70f2204c92943521/.ci/no-exception-test.sh#L44-L45

Reply:
> Checkstyle' -> Checkstyle

Done.